src/ws-week-picker/ws-week-picker.scss

```diff
@@ -0,0 +1,180 @@
+$icon-font-path: '../../node_modules/wholesale-styleguide-scss/icons/';
+@import "../../node_modules/wholesale-styleguide-scss/variables";
+@import "../../node_modules/wholesale-styleguide-scss/util";
+@import "../../node_modules/wholesale-styleguide-scss/components/base/icons";
+
+// PROPOSAL for mixin
+$padding: 1rem;
+// to allow padding with setting a transparent border
+@mixin table-padding ($top, $right, $bottom, $left) {
+  @if $top != 0 {
+    tr:first-child():before {
+      height: $top;
+    }
+  }
+  @if $right != 0 {
+    th:last-child(), td:last-child() {
+      padding-right: $right !important;
+    }
+  }
+  @if $bottom != 0 {
+    tr:first-child():after {
+      height: $bottom;
+    }
+  }
+  @if $left != 0 {
+    th:first-child(), td:first-child() {
+      padding-left: $left !important;
+    }
+  }
+}
+
+// PROPOSAL for style change
+@mixin arrow-tip-shadow {
+  box-shadow: 0 19px 38px rgba(0, 0, 0, 0.3), 0px 0px 9px rgba(0, 0, 0, 0.22);
+}
+
+.ws-week-picker {
+  position: relative;
+  display: inline-block;
+  $arrowSize: 10px;
+
+  & > input {
+    cursor: pointer;
+    display: inline-block;
+    padding-right: 1.5rem;
+  }
+  & > i {
+    position: absolute;
+    top: calc(0.5rem + 1px); // input has 1 pixel border (2px total)
+    right: 0.5rem;
+    cursor: pointer;
+  }
+  .ws-date-picker-calendar {
+    background: $athens-gray;
+    border-radius: 4px;
+    box-shadow: map-get($elevation-shadows, 'raised');
+    display: inline-block;
+    position: absolute;
+    top: 2.95rem;
+    left: 0;
+    z-index: map-get($z-layers, 'upper');
+    border: 1px solid $earl-gray;
+
+    &:before, &:after {
+      position: absolute;
+      display: block;
+      pointer-events: none;
+      border: solid transparent;
+      content: '';
+      height: 0;
+      width: 0;
+      border-width: 0.5rem;
+      margin: 0 -8px;
+      border-bottom-color: $earl-gray;
+      bottom: 100%;
+      left: 1rem;
+    }
+    &:after {
+      border-width: calc(0.5rem - 1px);
+      margin: 0 -7px;
+      border-bottom-color: $athens-gray;
+    }
+  }
+
+  table {
+    table-layout: fixed;
+    display: inline-block;
+    border: none;
+    font-size: $font-size-small;
+
+    caption {
+      padding: 0.5rem 0;
+      color: $heron;
+      font-weight: 300;
+      font-size: $base-font-size;
+
+      > span {
+        cursor: pointer;
+        line-height: 2rem;
```

Comment:
I'd make it 1.75rem; and just remove the height so the arrow align with the year